src/components/Navbar.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import logo from "../assets/images/kursi-logo.png";
+
+export const Navbar = () => {
+  return (
+    <header className="header py-0-5 z-1">
+      <nav className="flex align-center space-between wrap navbar-container">
+        <section className="brandlogo-wrapper pl-1 flex flex-center">
+          <a href="./index.html">
+            <img className="brandlogo-img py-0-5" src={logo} alt="Kursi" />
+          </a>
```

Comment:
Instead of "a href" you should use "Link" from react-router because using href will refresh the whole page each time you try to navigate. To overcome this issue, in react we use Link for the navigation.